camunda/external_task/external_task_executor.py

```diff
@@ -36,16 +36,28 @@ def _handle_task_result(self, task_result):
             self._log_with_context(err_msg, task_id=task_id, log_level='warning')
             raise Exception(err_msg)
 
+    def _strip_long_variables(self, variables):
+        """remove value of complex variables for the dict"""
+        if not variables:
+            return variables
+        cleaned = {}
+        for k, v in variables.items():
+            if isinstance(v, dict) and v.get("type", "") in ("File", "Bytes", "Json"):
```

Comment:
```suggestion
            if isinstance(v, dict) and v.get("type", "") in ("File", "Bytes"):
```
perhaps we still want the Json type to be logged.

Reply:
Ok. You are right. I will change it.